src/index.html

```diff
@@ -8,7 +8,11 @@
     <title>Stop watch</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Stop watch</h1>
+  <body class="body">
+    <div class="circle body__circle">
```

Comment:
[Checklist 5] Wrong BEM structure. You don't need body__circle class. To center your watch just add "margin: 0 auto" to the body tag